### DISCUSSION TITLE
How do you switch the main language?

### DISCUSSION THREAD

**Discussion by joshclancy** [tuna/thuthesis#640] [Q&A] [answered]
I see all of this code in the .bst file for changing the main language of the document. Where is the initial variable set? How can I change the main language to English?

**Comment by zepinglee** [comment, marked as answer]
`\documentclass[degree=master, language=english]{thuthesis}`

The `.bst` files are used to format the bibliography with BibTeX.

**Comment by joshclancy** [comment]
Thank you very much! Should have guessed that.
